src/applications/pokemon_summary_screen/main.c

```diff
@@ -564,27 +608,27 @@ static int HandleInput_Main(PokemonSummaryScreen *summaryScreen)
 
         if (summaryScreen->page == PSS_PAGE_BATTLE_MOVES) {
             Sound_PlayEffect(SEQ_SE_DP_SYU01);
-            summaryScreen->subscreen = 0;
+            summaryScreen->pageState = 0;
             return PSS_STATE_SETUP_BATTLE_MOVE_INFO;
         } else if (summaryScreen->page == PSS_PAGE_CONTEST_MOVES) {
             Sound_PlayEffect(SEQ_SE_DP_SYU01);
-            summaryScreen->subscreen = 0;
+            summaryScreen->pageState = 0;
             return PSS_STATE_SETUP_CONTEST_MOVE_INFO;
         } else if (summaryScreen->page == PSS_PAGE_RIBBONS) {
             if (summaryScreen->ribbonMax != 0) {
                 Sound_PlayEffect(SEQ_SE_DP_DECIDE);
-                summaryScreen->subscreen = 0;
+                summaryScreen->pageState = 0;
                 return PSS_STATE_SETUP_RIBBON_INFO;
             }
         } else if (summaryScreen->page == PSS_PAGE_EXIT) {
             Sound_PlayEffect(SEQ_SE_DP_DECIDE);
-            summaryScreen->data->returnMode = 1;
+            summaryScreen->data->returnMode = PSS_RETURN_CANCEL;
             return PSS_STATE_TRANSITION_OUT;
         }
     }
 
     if (CheckSubscreenPressAndSetButton(summaryScreen) == TRUE) {
-        summaryScreen->subscreen = 2;
+        summaryScreen->pageState = 2;
```

Comment:
**question**: Do we have enough info to document these constants?

Reply:
Yep, added some documentation for them.